libaftermath-core/src/defs/aftermath/types/openmp/on_disk.py

```diff
@@ -250,6 +250,42 @@
 
 ################################################################################
 
+am_dsk_openmp_mutex_released = EventFrame(
+    name = "am_dsk_openmp_mutex_released",
+    entity = "on-disk OpenMP mutex released event",
+    comment = "Event signalizing release of a mutex",
+    fields = FieldList([
+        Field(
+            name = "timestamp",
+            field_type = aftermath.types.builtin.uint64_t,
+            comment = "Time of the event"),
+        Field(
+            name = "wait_id",
+            field_type = aftermath.types.builtin.uint64_t,
+            comment = "ID of the object that is bing awaited"),
+        Field(
+            name = "kind",
+            field_type = aftermath.types.builtin.uint32_t,
+            comment = "Type of the mutual exclusion event")]))
+
+################################################################################
+
+am_dsk_openmp_mutex_type = Frame(
+    name = "am_dsk_openmp_mutex_type",
+    entity = "on-disk OpenMP mutex type",
+    comment = "Association between type id and type name",
+    fields = FieldList([
+        Field(
+            name = "type_id",
+            field_type = aftermath.types.builtin.uint32_t,
+            comment = "ID of the mutex type"),
+        Field(
+            name = "name",
+            field_type = aftermath.types.on_disk.am_dsk_string,
+            comment = "Human readable description of mutex type")]))
```

Comment:
Is this supposed to be just a name or a complete description? If it's just a name, then human readable _name_ would be more accurate for the comment.

Reply:
Resolved after commits got squashed.